tree/dataframe/src/RLoopManager.cxx

```diff
@@ -501,12 +533,13 @@ void RLoopManager::RunTreeProcessorMT()
 /// Run event loop over one or multiple ROOT files, in sequence.
 void RLoopManager::RunTreeReader()
 {
+   if (fEndEntry == fStartEntry) // empty range => no work needed
```

Comment:
When does this happen? The default values are 0 and max

Reply:
Is this only there if the user explicitly writes e.g. `{0, 0}` to the RDatasetSpec constructor?

Reply:
well, `fEndEntry` and `fStartEntry` are taken from the spec. As previously discussed, might provide an empty range.

On the one hand, there might exist an empty TTree => no work.

On the other hand, current implementation of `Range` would error out if an empty range is provided.

I currently do the first option, e.g. explicity no work.

The reason why I believe we need this extra check is: if I remove this case handling, in the subsequent lines `SetEntriesRange(fStartEntry, fEndEntry)` will take all entries.

So what do we want?
1. Do no work? --> current
2. Do all work? --> if I remove the `if (fEndEntry == fStartEntry)`
3. Error out? --> simply throw when  fEndEntry == fStartEntry